### DISCUSSION TITLE
Pixel 7 Compatibility

### DISCUSSION THREAD

**Discussion by dactylroot** [zeapo/Android-Password-Store#2541] [General]
After switching to a Pixel 7, I noticed the application wasn't working correctly. Attempting to reinstall, I now find "This app is not available for any of your devices" from the Google store. Is this happening for anyone else?

**Comment by msfjarvis** [comment]
That's expected, the app targets a too old version of Android which Google Play rejects on new devices for safety and privacy reasons.

The development branch of the repository contains a work-in-progress rewrite of the app with significant UI and cryptography changes which will work on the Pixel 7 but may have UX problems and bugs. You can get a build from the development branch through the [snapshot releases](https://docs.passwordstore.app/docs/users/release-channels).
